settings/settings.go

```diff
@@ -204,15 +191,51 @@ func AddPools(containerEngine airflow.ContainerHandler, id string, airflowVersio
 				} else {
 					airflowCommand += ""
 				}
-				containerEngine.ExecCommand(id, airflowCommand)
-				fmt.Printf("Added Pool: %s\n", pool.PoolName)
+				_, err := containerEngine.ExecCommand(id, airflowCommand)
+				if err != nil {
+					fmt.Printf("Error adding pool %s: %s\n", pool.PoolName, err.Error())
+				} else {
+					fmt.Printf("Added Pool: %s\n", pool.PoolName)
+				}
 			} else {
 				fmt.Printf("Skipping %s: Pool Slot must be set.\n", pool.PoolName)
 			}
 		}
 	}
 }
 
+func prepareAddConnCmd(baseAddCmd, connIDArg, connTypeArg, connURIArg, connExtraArg, connHostArg, connLoginArg, connPasswordArg, connSchemaArg, connPortArg string, conn *Connection) string {
```

Comment:
I feel like this logic could be reworked, we have many different string variables being initialized and passed around.

I believe we can affort to wait for the CLI Convergence to fix/clean the code in the repository as we go through the migration.

Reply:
+1 to rework this part.